### PR TITLE
Increase branch coverage

### LINKED ISSUE #92: Circle CI Build is failing
Need to raise the branch coverage from ~78 to above the threshold which is 80

### PR DESCRIPTION
**Pull request by p-netm**
Master build is failin since branch coverage is wanting, this Tries to fix that.
closes #92

### CODE REVIEW

File: src/configs/env.ts [hunk 1]
```diff
@@ -83,32 +83,6 @@ export type OPENSRP_USER_URL = typeof OPENSRP_USER_URL;
...
-export const ONADATA_USER_URL =
-  process.env.REACT_APP_ONADATA_USER_URL || 'https://stage-api.ona.io/api/v1/user.json';
-export type ONADATA_USER_URL = typeof ONADATA_USER_URL;
-
-export const ONADATA_OAUTH_STATE = process.env.REACT_APP_ONADATA_OAUTH_STATE || 'onadata';
-export type ONADATA_OAUTH_STATE = typeof ONADATA_OAUTH_STATE;
-
-export const GISIDA_ONADATA_API_TOKEN = process.env.REACT_APP_GISIDA_ONADATA_API_TOKEN || '';
-export type GISIDA_ONADATA_API_TOKEN = typeof GISIDA_ONADATA_API_TOKEN;
-
```
Comment (FrankApiyo) [inline src/configs/env.ts]: alright.

File: src/components/page/SideMenu/SubMenu/tests/index.test.tsx [hunk 2]
```diff
@@ -111,4 +111,48 @@ describe('components/page/SubMenu', () => {
     expect(mockCallBack.mock.calls.length).toEqual(1);
     wrapper.unmount();
   });
+
+  it('stimulates click and calls mock function with correct value', () => {
```
Comment (FrankApiyo) [inline src/components/page/SideMenu/SubMenu/tests/index.test.tsx]: you meant `simulate` here I think.

**Comment by p-netm** [comment]
Decided to revert on the decision to remove onadata oauth, i think there would need for a team discussion if its something that should still remain in the codebase even though IMO its not currently used.

File: src/components/page/SideMenu/SubMenu/tests/index.test.tsx (hunk 2 above)
Comment (p-netm) [reply src/components/page/SideMenu/SubMenu/tests/index.test.tsx]: yeah

File: src/configs/env.ts (hunk 1 above)
Comment (p-netm) [reply src/configs/env.ts]: Went back on this decision.

Comment (FrankApiyo) [reply src/configs/env.ts]: These code that is not in use should totally be removed.

Comment (p-netm) [reply src/configs/env.ts]: @hmakandi @cKellyDesign Do you think we should keep the oauth settings for onadata as a provider?

Comment (cKellyDesign) [reply src/configs/env.ts]: @p-netm @FrankApiyo - Is OpenSRP able to authenticate with Ona's oAuth provider? If so then it should remain an option, if not then it should be removed.

**Pull request merged**